reference-architecture/aws-ansible/ose-on-aws.py

```diff
@@ -76,9 +76,12 @@
 @click.option('--github-client-secret', help='GitHub OAuth Client Secret')
 @click.option('--github-organization', multiple=True, help='GitHub Organization')
 @click.option('--s3-username',  help='S3 user for registry access')
-@click.option('--deploy-openshift-metrics',  help='Deploy OpenShift Metrics', type=click.Choice(['true', 'false']), default='false')
+@click.option('--openshift-hosted-metrics-deploy',  help='Deploy OpenShift Metrics', type=click.Choice(['true', 'false']), default='false')
+@click.option('--openshift-hosted-logging-deploy',  help='Deploy OpenShift Logging', type=click.Choice(['true', 'false']), default='false')
```

Comment:
I'd say to remove the "hosted" word as it doesn't say anything besides typing more :D and make both logging and metrics enabled by default